WowUp.WPF/Services/AddonService.cs

```diff
@@ -369,7 +388,14 @@ private void RemoveThumbnail(Addon addon)
                 targetAddonChannel = latestFile.ChannelType;
             }
 
-            return CreateAddon(latestFile.Folders.FirstOrDefault(), searchResult, latestFile, clientType, targetAddonChannel);
+            foreach (var dependency in latestFile.Dependencies.Where(dep => dep.Type == AddonDependencyType.Required))
```

Comment:
needs a '?' tukui and wowi addons wont have deps

Reply:
I implemented the Null object pattern for the dependencies to all providers, but somehow i did not commit those changes, resolved it now.